apps/ui/src/stores/meta.ts

```diff
@@ -44,6 +44,11 @@ export const useMetaStore = defineStore('meta', () => {
     return Math.round(current * METADATA[networkId].blockTime);
   }
 
+  function getDurationFromCurrentEVM(networkId: NetworkID, current: number) {
+    if (!evmNetworks.includes(networkId)) return current;
+    return getDurationFromCurrent(networkId, current);
+  }
```

Comment:
There is something not right with this, if should be automated by the use of `currentUnit` so we shouldn't need any more extra special cases, we should check why it doesn't work like we expect and if there is bigger issue fix it generally as we use `getDurationFromCurrent` in different places as well.

Is this still needed?

Testing all 3 types:
- http://localhost:8080/#/matic:0xcD4c00cAB0A86C182519e91a8b01b080B484C49B/create
- http://localhost:8080/#/s:0cf5e.eth/create
- http://localhost:8080/#/sn:0x05702362b68a350c1cae8f2a529d74fdbb502369ddcebfadac7e91da37636947/create
- http://localhost:8080/#/matic:0xcD4c00cAB0A86C182519e91a8b01b080B484C49B/proposal/1
- http://localhost:8080/#/s:0cf5e.eth/proposal/0x0a04e2d340fcd5cfcac5d89369d152c8c9fefc59126c48b650785c147dc3d6aa
- http://localhost:8080/#/sn:0x05702362b68a350c1cae8f2a529d74fdbb502369ddcebfadac7e91da37636947/proposal/14

All looks good with default `getDurationFromCurrent`.

Reply:
Ahh, great point, magic is here 
https://github.com/snapshot-labs/sx-monorepo/blob/master/apps/ui/src/stores/meta.ts#L34
It is `second` for off chain and sn but `block` for EVM, so `getDurationFromCurrent` should work everywhere 😅 
<img width="391" alt="Untitled 2" src="https://github.com/user-attachments/assets/11e840d2-5441-4610-876c-8c98e96b72d8">

Reply:
Changed it to use `getDurationFromCurrent`